client/src/App.js

```diff
@@ -57,8 +58,10 @@ function App() {
 					<Route path="/about/events" component={OurEvents} />
 
 					<Route exact path="/resources" component={Resources} />
-          			<Route path="/resources/timetables" component={Timetables}/>
-
+          <Route path="/resources/timetables" component={Timetables}/>
```

Comment:
Hey Andrejs 
please rename Folder to "Timetables" (capital T at the front), so that i woul be consistent with all other folder names with the project.